File: JAVA/3184. Count Pairs That Form a Complete Day I.java
class Solution {
    public int countCompleteDayPairs(int[] h) {
        int c=0,i,j;
        for(i=0;i<h.length;i++){
            for(j=i+1;j<h.length;j++){
                if((h[i]+h[j])%24==0){
                    c++;
                }            
        }
    }
    return c;
    }
}

File: C++/3184. Count Pairs That Form a Complete Day I.cpp
class Solution {
public:
    int countCompleteDayPairs(vector<int>& h) {
        int c=0,i,j;
        for(i=0;i<h.size();i++){
            for(j=i+1;j<h.size();j++){
                if((h[i]+h[j])%24==0)
                c++;
            }
        }
        return c;
    }
};